Apps/Contoso.iOS.Puppet/ModulePages/AppCenterController.cs

```diff
@@ -83,5 +91,16 @@ public override void PrepareForSegue(UIStoryboardSegue segue, NSObject sender)
             string tag = LogWriteTag.Text;
             LogFunctions[mLogWriteLevel](tag, message);
         }
+
+        partial void SaveStorageSize(NSObject sender)
+        {
+            var inputText = StorageSizeText.Text;
+            if (int.TryParse(inputText, out var size))
```

Comment:
and if we didn't parse text successfully what will happen? Any notification to user or log line?